python/ray/autoscaler/ray-schema.json

```diff
@@ -24,7 +24,7 @@
             "type": "string"
         },
         "max_workers": {
-            "description": "The maximum number of workers nodes to launch in addition to the head node. This takes precedence over min_workers.",
+            "description": "The maximum number of workers nodes to launch in addition to the head node. This should be no larger than the sum of min_workers for all available node types.",
```

Comment:
We actually check ^ in our existing codebase:

https://github.com/ray-project/ray/blob/a77751707b528ecd3e55f580c415833d4179f4f1/python/ray/autoscaler/_private/util.py#L177-L193